apps/ledger-live-desktop/tests/specs/syncOnboarding/install-set-of-apps.spec.ts

```diff
@@ -0,0 +1,91 @@
+import { ManagerPage } from "../../models/ManagerPage";
+
+test.use({ userdata: "skip-onboarding" });
+
+test("Install set of apps", async ({ page }) => {
+  const settingsPage = new SettingsPage(page);
+  const layout = new Layout(page);
+  const deviceAction = new DeviceAction(page);
+  const installSetOfAppsPage = new InstallSetOfApps(page);
+  const managerPage = new ManagerPage(page);
+
+  await test.step("go to debug install set of apps drawer", async () => {
+    /*await layout.goToManager();
+    await deviceAction.accessManager();
+    await managerPage.firmwareUpdateButton.waitFor({ state: "visible" });*/
+    await layout.goToSettings();
+    await settingsPage.goToAboutTab();
+    await settingsPage.enableAndGoToDeveloperTab();
+    await installSetOfAppsPage.waitForLaunch();
+    await installSetOfAppsPage.accessDebugInstallSetOfApps();
+    await expect(page).toHaveScreenshot("debug-install-set-of-apps-drawer.png");
+  });
+
+  await test.step("use default apps", async () => {
+    await installSetOfAppsPage.startInstallOrRestoreSetOfApps();
+    await expect(page).toHaveScreenshot("debug-install-set-of-apps-queue.png");
+    await deviceAction.installSetOfAppsMocked(0.25, 0.12, { type: "install", name: "Bitcoin" }, [
+      "Bitcoin",
+      "Ethereum",
+      "Polygon",
+    ]);
+    await page.waitForTimeout(100);
+    await expect(page).toHaveScreenshot("debug-install-set-of-apps-step-1.png");
+    await deviceAction.installSetOfAppsMocked(0.5, 0.5, { type: "install", name: "Ethereum" }, [
+      "Ethereum",
+      "Polygon",
+    ]);
+    await page.waitForTimeout(100);
+    await expect(page).toHaveScreenshot("debug-install-set-of-apps-step-2.png");
+    await deviceAction.installSetOfAppsMocked(0.87, 0.75, { type: "install", name: "Polygon" }, [
+      "Polygon",
+    ]);
+    await page.waitForTimeout(100);
+    await expect(page).toHaveScreenshot("debug-install-set-of-apps-step-3.png");
+    await deviceAction.mockOpened();
+    await page.waitForTimeout(2500);
+    await expect(page).toHaveScreenshot("debug-install-set-of-apps-completed.png");
+  });
+
+  await test.step("restore apps from Ledger Stax", async () => {
+    await installSetOfAppsPage.resetButtonClick();
+    await expect(page).toHaveScreenshot("debug-install-set-of-apps-reseted.png");
+    await deviceAction.installSetOfAppsMocked(0, 0, { type: "install", name: "Bitcoin" }, [
+      "Ethereum",
+      "Polygon",
+    ]);
+    await page.waitForTimeout(100);
+    await installSetOfAppsPage.startRestoreAppsFromStax();
+    await expect(page).toHaveScreenshot("debug-restore-set-of-apps-drawer.png");
+    await installSetOfAppsPage.startInstallOrRestoreSetOfApps();
+    await expect(page).toHaveScreenshot("debug-restore-set-of-apps-queue.png");
+    await deviceAction.installSetOfAppsMocked(0.3, 0.15, { type: "install", name: "Ethereum" }, [
+      "Ethereum",
+      "Polygon",
+    ]);
+    await page.waitForTimeout(100);
```

Comment:
Why are you waiting for a timeout here (and below)?
It's not recommended as the timeout is an arbitrary value so it can bring flakiness. If possible rely on something that comes from the UI.

Reply:
We need a small delay because there is a delay on the UI after the mock of stream-install. There is a longer delay with the mock of open. I will find another solution to remove the delay.